src/frontend/planner_test/tests/testdata/expr.yaml

```diff
@@ -351,3 +351,22 @@
   batch_plan: |
     BatchProject { exprs: ['integer[][]':Varchar] }
     └─BatchValues { rows: [[]] }
+- name: now expression
+  sql: |
+    create table t (v1 timestamp);
+    select * from t where v1 >= now();
+  logical_plan: |
+    LogicalProject { exprs: [t.v1] }
+    └─LogicalFilter { predicate: (t.v1 >= Now) }
+      └─LogicalScan { table: t, columns: [t.v1, t._row_id] }
+  stream_plan: |
+    StreamMaterialize { columns: [v1, t._row_id(hidden)], pk_columns: [t._row_id] }
+    └─StreamDynamicFilter { predicate: (t.v1 >= t.v1), output: [t.v1, t._row_id] }
```

Comment:
> predicate: (t.v1 >= t.v1)

Doesn't seem to be correct?

Reply:
> > predicate: (t.v1 >= t.v1)
> 
> Doesn't seem to be correct?

It's correct.
It's just a `StreamDynamicFilter` display bug.